examples/go-chart/main.go

```diff
@@ -78,19 +82,19 @@ func main() {
 
 	f, _ := os.Create("output.pdf")
 	defer f.Close()
-	graph.Render(canvas.NewGoChart(canvas.OutputPDF), f)
+	graph.Render(canvas.NewGoChart(pdf.Writer), f)
 
 	f, _ = os.Create("output.svg")
 	defer f.Close()
-	graph.Render(canvas.NewGoChart(canvas.OutputSVG), f)
+	graph.Render(canvas.NewGoChart(svg.Writer), f)
 
 	f, _ = os.Create("output.eps")
 	defer f.Close()
-	graph.Render(canvas.NewGoChart(canvas.OutputEPS), f)
+	graph.Render(canvas.NewGoChart(eps.Writer), f)
 
 	f, _ = os.Create("output.png")
 	defer f.Close()
-	graph.Render(canvas.NewGoChart(canvas.OutputPNG), f)
+	graph.Render(canvas.NewGoChart(rasterizer.PNGWriter(1.0)), f)
```

Comment:
Could we clarify that `1.0` is in `DPM` somehow? Something alike `72 * DPI`...?

Reply:
Do you want me to update the examples to
```go
graph.Render(canvas.NewGoChart(rasterizer.PNGWriter(canvas.DPPM(1.0))), f)
```
I could also simply add a comment, since most of the people will use it like this:
```go
graph.Render(canvas.NewGoChart(rasterizer.PNGWriter(1.0)), f) // 1.0 is the resolution, in dot-per-millimeter
```

Reply:
Let's leave it as it is for now